taglib/mp4/mp4atom.cpp

```diff
@@ -131,6 +124,8 @@ MP4::Atom::Atom(File *file)
     }
   }
 
+  debug("MP4: Skipping atom '" + d->name + "'");
+
```

Comment:
This will print a lot of atom names while parsing and possibly disturb the output because `d->name` can be anything. Therefore I would remove this.

Reply:
You're right, even in my testing it was a bit annoying